src/hash-table.js

```diff
@@ -7,6 +7,20 @@ class HashTable {
     this.storage = new LimitedArray(this.limit);
     // Do not modify anything inside of the constructor
   }
+  insert(key, value) {
+    const index = getIndexBelowMax(key.toString(), this.limit);
+    this.storage.set(index, value);
+    if (this.storage.length >= 6) {
```

Comment:
So it seems like y'all are hardcoding the scale, consider doing it on a scaling increase (what if I hit 12/16 on storage?)